src/compiler/data.ts

```diff
@@ -18,7 +19,7 @@ import {NOMINAL, ORDINAL, QUANTITATIVE, TEMPORAL} from '../type';
  *                 If the encoding contains aggregate value, this will also create
  *                 aggregate table as well.
  */
-export default function(model: Model) {
+export function def(model: Model) {
```

Comment:
We shouldn't use `default` because it creates weird `.default` when you do require in the compiled javascript.  But I do agree that `def` isn't the right way to name this.